both/collections/schemas/user.js

```diff
@@ -21,6 +21,39 @@ Schemas.userWarnings = new SimpleSchema({
   }
 });
 
+Schemas.class = new SimpleSchema({
+  title: (new SimpleSchemaFields('user class')).title(),
+  shortTitle: {
```

Comment:
Why don't you use SimpleSchemaFields for the short title as well? Also why not use them in Schema.userClass as well?